src/commonTest/kotlin/fr/acinq/lightning/wire/RouteBlindingTestsCommon.kt

```diff
@@ -14,7 +15,7 @@ class RouteBlindingTestsCommon : LightningTestSuite() {
             ByteVector("01080000000000000000 042102edabbd16b41c8371b92ef2f04c1185b4f03b6dcd52ba9b78d9d7c89c8f221145") to RouteBlindingEncryptedData(
                 TlvStream(
                     Padding(ByteVector("0000000000000000")),
-                    OutgoingNodeId(PublicKey(ByteVector("02edabbd16b41c8371b92ef2f04c1185b4f03b6dcd52ba9b78d9d7c89c8f221145")))
+                    OutgoingNodeId(EncodedNodeId(PublicKey(ByteVector("02edabbd16b41c8371b92ef2f04c1185b4f03b6dcd52ba9b78d9d7c89c8f221145"))))
```

Comment:
Actually it would be useful to have an encoding test that uses the scid_dir format, that we would share with eclair (and maybe even the spec).

Reply:
I've added a test, here and in eclair too (https://github.com/ACINQ/eclair/pull/2821)

Reply:
It looks like you didn't push that commit though, I don't see any test...

Reply:
Indeed, pushed just now